index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="query.css">
+  <script src="https://kit.fontawesome.com/43fbd35f63.js" crossorigin="anonymous"></script>
+  <title>Microverse Capstone Project</title>
+</head>
+<body>
+  <!-- Project Header -->
+  <header>
+    <!-- Project Navbar -->
+    <nav class="navbar">
+      
+        <a aria-label="menu" href="#home" class="hamburger-menu"><i id="menu-open" class="fas fa-bars"></i></a>
+      <div class="menu-desktop-logo">
+        <div class="logo">
+          <a href="#home"><img src="src/project-logo.webp" alt="logo"></a>
```

Comment:
- [ ] I noticed that navigating back to the index page from the about page doesn't work on desktop view.
I think the problem is in the `href="#home"` Try using this `href="index.html"`.

Reply:
ok but this is the hamburger menu of the index.html. i have to link the menu to the same page ?

Reply:
Inside the desktop menu, there is a link for the home page